Update rules.md

veena-vee989 opened the pull request:
@beecastanon @natestemen I've added in some language about probationary participation. 

I'm not sure if we need to include specifics here primarily because the Rules page is quite short, and I don't want the disciplinary action section to be the majority of it. That seems like we're starting off on a negative note.. 

Let me know what you both think though!

nathanshammah reviewed (approved):
Thanks @josh146, LGTM!

crazy4pi314 reviewed (approved):
LGTM, Thanks for adding the last one 💖

veena-vee989 committed:
Update rules.md

@beecastanon @natestemen I've added in some language about probationary participation. 

I'm not sure if we need to include specifics here primarily because the Rules page is quite short, and I don't want the disciplinary action section to be the majority of it. That seems like we're starting off on a negative note.. 

Let me know what you both think though!

veena-vee989 committed:
Update rules.md

veena-vee989 committed:
Update rules.md

veena-vee989 committed:
Update rules.md

Added a bit more language based on suggestions from Ben
